containers/local.env

```diff
@@ -15,6 +17,10 @@ FILES_DB_BUCKET=testbucket
 FILES_DB_ACCESS_KEY=minioadmin
 FILES_DB_SECRET_KEY=minioadmin
 
+STRIPE_SECRET="${STRIPE_SECRET:-value from stripe dashboard}"
```

Comment:
what is this `${}` business?

Reply:
If the stripe secret is not already defined, then we will give it a default value. This is important to people who want to have their own value if they are testing stripe.

Reply:
what tool processes this? Is this a `zsh` thing, or is it available under `sh`? Will this work with `docker` in CI? We'll probably find out that bit when you fix the merge conflicts